gtfs/feed_sources/Berlin.py

```diff
@@ -0,0 +1,9 @@
+"""Fetch Berlin feed."""
+from utils.FeedSource import FeedSource
```

Comment:
you can put the `FeedSource.py` into the `gtfs` root IMO, it's not really a "util". also file names rather lower case, i.e. `feed_source.py`